pkg/bridge/ai/api_server.go

```diff
@@ -20,10 +20,12 @@ const (
 	DefaultZipperAddr = "localhost:9000"
 )
 
-// RequestTimeout is the timeout for the request, default is 5 seconds
-var RequestTimeout = 5 * time.Second
-
-// ======================= Interface =======================
+var (
+	// RequestTimeout is the timeout for the request, default is 90 seconds
+	RequestTimeout = 90 * time.Second
+	//  RunFunctionTimeout is the timeout for awaiting the function response, default is 60 seconds
+	RunFunctionTimeout = 60 * time.Second
```

Comment:
```go
	RequestTimeout = 90 * time.Second
	RunFunctionTimeout = 60 * time.Second
```
Recommended move this two variables to `github.com/yomorun/yomo/ai` package, so developers can modify.

Reply:
These two global variables are placed in `github.com/yomorun/yomo/pkg/bridge/ai` now, I think this is better.